src/test/java/api/loans/DeclareItemClaimedReturnedAsMissingApiTests.java

```diff
@@ -0,0 +1,123 @@
+package api.loans;
+
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static api.support.matchers.LoanMatchers.hasLoanProperty;
+import static api.support.matchers.LoanMatchers.isClosed;
+import static api.support.matchers.ValidationErrorMatchers.hasErrorWith;
+import static api.support.matchers.ValidationErrorMatchers.hasMessage;
+import static api.support.matchers.ValidationErrorMatchers.hasParameter;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION_COMMENT;
+import static org.folio.circulation.resources.DeclareItemClaimedReturnedAsMissingResource.COMMENT;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.ClaimItemReturnedRequestBuilder;
+import api.support.builders.DeclareItemClaimedReturnedAsMissingRequestBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+public class DeclareItemClaimedReturnedAsMissingApiTests extends APITests {
```

Comment:
```suggestion
public class DeclareClaimedReturnedItemMissingApiTests extends APITests {
```

Reply:
API endpoint has been renamed as well to `declare-claimed-returned-item-as-missing`.